picasso/helper.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+"""helper code to provide the Flask server with information
+
+This code only provides helper functions.
+"""
+from types import ModuleType
+from importlib import import_module
+from picasso.visualizations.base import BaseVisualization
+import inspect
+from flask import (
+    g,
+    current_app
+)
+from picasso.models.base import load_model
+
+
+class Helper:
```

Comment:
I like the idea of splitting these helper functions out into a separate file.  We don't need a new class here though (all the members are static!) -- can you just import the functions into `picasso.py` directly?  And rename `helper.py` to `utils.py` or preferably something even more descriptive.  "helpers" is a bit too generic.